plugins/fusion/covert.py

```diff
@@ -21,9 +21,12 @@
 
 from electroncash.util import PrintError
 
-def is_tor_port(port):
+def is_tor_port(port, host=None):
```

Comment:
Can you tweak this to just be `is_tor_port(host, port)` and change the callsites appropriately? Seems to me that explicit is best in this case.

Reply:
Will do.